README.md

```diff
@@ -1,3 +1,317 @@
 # mbq.apitools
 
-hi travis!
+`mbq.apitools` is a python library for writing endpoints in Django. It provides a view decorator and view class that allow for strict typing of incoming query parameters and payloads, as well as consistent response shapes and status codes on the way out.
+
+Some nice things about `mbq.apitools`:
+* All fields specified in a param or payload schema are required by default, and can be marked as optional by providing a `default=` argument to the field class. The framework will automatically return a 400 response for all requests which do not conform to the specified schema. Details for each nonconforming field will be included in the response.
+* The parsed parameters and payloads end up as rich types on the request. If a field is marked as `fields.DateTime`, then it will be on `request.payload` as a `datetime` object.
+* Pagination is handled entirely by the framework. Just mark a view as paginated and return an instance of a `PaginatedResponse` and voila!
```

Comment:
"mark a view as paginated" is a little ambiguous. Might be worth saying:


"Simply include `paginated=True` when you define the view, return an instance of a `PaginatedResponse` and voila!"